lib/gems/pending/util/postgres_admin.rb

```diff
@@ -249,7 +249,7 @@ def self.runcmd_with_logging(cmd_str, opts, params = {})
     {
       "PGUSER"     => opts[:username],
       "PGPASSWORD" => opts[:password]
-    }
+    }.delete_if { |n, v| v.blank? }
```

Comment:
Do we have `delete_blanks` here? I can't remember if that's activesupport or morecore ...

Also, can you fix the code climate/bot comment about the unused argument?

Reply:
`more_core_extensions` is in the gemspec, so it should be good. (I always avoid morecore but in this case, it is good. thnx)